src/Tribe/Service_Providers/Context.php

```diff
@@ -370,11 +373,11 @@ static function () {
 						],
 					],
 				],
-				'view_request'         => [
+				'view_request'       => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 9 space(s) between "'view_request'" and double arrow, but found 7.

<sub>hash: ae3da0cdac493a1f40cdd0bcbe51c49e</sub>